modules/dynamic-plug-in-development.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins-get-started.adoc
+
+:_content-type: PROCEDURE
+[id="dynamic-plugin-development_{context}"]
+= Dynamic plug-in development
+
+You can run the plug-in using a local development environment. The {product-title} web console runs in a container connected to the cluster you have logged into.
+
+.Prerequisites
+* You must have an OpenShift cluster running.
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have link:https://yarnpkg.com/[`yarn`] installed.
+* You must have link:https://www.docker.com/[Docker] v3.2.0 or newer or link:https://podman.io/[Podman] installed.
+
+.Procedure
+
+. In your terminal, run the following command to install yarn.
+
++
+[source,terminal]
+----
+yarn install
```

Comment:
Shall we clarify user should be in the local forked console-plugin-template folder to run all following commands?

Reply:
I am not sure we are able to do this here. When Sam and I talked to support, the only way we are able to link to the template repo is with an example. I have added this page as an additional resource at the beginning of the "Adding a tab to the pods page" example.  cc @spadgett